Hackney.Shared.Tenure.Tests/Boundary/Validation/TenureInformationValidatorWhenOnlyEndDateTests.cs

```diff
@@ -10,6 +10,8 @@ public class TenureInformationValidatorWhenOnlyEndDateTests
     {
         public TenureInformationValidatorWhenOnlyEndDate _classUnderTest;
 
+        DateTime _now = new DateTime(2024, 5, 7);
```

Comment:
Not exactly `"now"` anymore as it's hardcoded, but I suppose it makes no functional difference to the test.

Could rename the variable to `startDate` or something among these lines.

Reply:
that's true, I've renamed it throughout.